client/landing/gutenboarding/components/signup-form/index.tsx

```diff
@@ -101,46 +102,55 @@ const SignupForm = ( { onRequestClose }: Props ) => {
 
 	return (
 		<Modal
-			className="signup-form"
-			title={ NO__( 'Sign up to save your changes' ) }
+			className={ 'signup-form' }
+			title={ NO__( 'Save your progress' ) }
 			onRequestClose={ closeModal }
 			focusOnMount={ false }
-			isDismissible={ ! isFetchingNewUser }
+			isDismissible={ false }
```

Comment:
Not sure about what `isDismissible={ false }` is doing because pressing Esc is definitely still working. We do what the modal dismissible right?

Reply:
Oh, this is a hack that just ensures that the `Modal` component's default close button isn't rendered. We then use our own. I think `isDismissible` is more accurately a `showCloseButton` feature.

Reply:
Ah! Thanks for that. I do think that you showing our own header, rather than bending over backwards with hacky CSS change the existing one do what we want, is a good idea.